src/FFMpeg/Media/Video.php

```diff
@@ -121,11 +123,9 @@ public function save(FormatInterface $format, $outputPathfile)
         }
 
         // If the user passed some additional parameters
-        if ($format instanceof VideoInterface) {
-            if (null !== $format->getAdditionalParameters()) {
-                foreach ($format->getAdditionalParameters() as $additionalParameter) {
-                    $commands[] = $additionalParameter;
-                }
+        if (null !== $format->getAdditionalParameters()) {
+            foreach ($format->getAdditionalParameters() as $additionalParameter) {
+                $commands[] = $additionalParameter;
             }
         }
```

Comment:
Shouldn't we put this:

```php
if (null !== $format->getAdditionalParameters()) {
             foreach ($format->getAdditionalParameters() as $additionalParameter) {
                 $commands[] = $additionalParameter;
              }
          }
```
inside the test `if ($format instanceof AudioInterface)` ?